packages/manager/modules/hub/src/components/billing-summary/component.js

```diff
@@ -0,0 +1,13 @@
+import controller from './controller';
+import template from './template.html';
+
+export default {
+  controller,
+  template,
+  bindings: {
+    me: '<?',
+    bills: '<?',
+    debt: '<?',
+    goToBills: '<',
```

Comment:
Guess the component can internally handle the redirection, since the target url is not going to changeif passed into the component or if handled by the component itself :)
What do you think about it?

Reply:
yes i'm waiting for https://github.com/ovh/manager/pull/2225 to be merged and i'll refactor it with RedirectionService